src/index.ts

```diff
@@ -13,23 +13,20 @@ async function run() {
     const architecture = process.arch;
     const workspaceDir = process.cwd();
 
+    // Export env variable to inject the buildtool plugin
+    core.exportVariable('MW_MATLAB_BUILDTOOL_DEFAULT_PLUGINS_FCN_OVERRIDE', 'matlab.ciplugins.github.getDefaultPlugins');
+
     const options: buildtool.RunBuildOptions = {
         Tasks: core.getInput("tasks"),
         BuildOptions: core.getInput("build-options"),
     };
 
+    core.info("Running MATLAB Build");
```

Comment:
Not sure I see much value in the message given that the entire task is about running the MATLAB build.

Reply:
I feel it is quite informative. However, I will remove it as I see @davidbuzinski is too agree